js/binance.js

```diff
@@ -4142,19 +4143,54 @@ module.exports = class binance extends Exchange {
          */
         await this.loadMarkets ();
         const currency = this.currency (code);
-        let type = this.safeString (params, 'type');
-        if (type === undefined) {
-            const accountsByType = this.safeValue (this.options, 'accountsByType', {});
-            const fromId = this.safeString (accountsByType, fromAccount, fromAccount);
-            const toId = this.safeString (accountsByType, toAccount, toAccount);
-            type = fromId + '_' + toId;
-        }
         const request = {
             'asset': currency['id'],
             'amount': this.currencyToPrecision (code, amount),
-            'type': type,
         };
-        const response = await this.sapiPostAssetTransfer (this.extend (request, params));
+        request['type'] = this.safeString (params, 'type');
+        let method = 'sapiPostAssetTransfer';
+        if (request['type'] === undefined) {
```

Comment:
@samgermain currently if we try to do this:
![image](https://user-images.githubusercontent.com/43336371/187026217-ee3cd5c0-09de-43f5-9e70-7e6ad6df097c.png)


we crash with a very generic error,  do you think we could improve this by either crashing before making the request with a very clear error message or supporting something like account='isolated' + symbol in params? What do you think?